src/FlowerShop.Core/Entities/Order.cs

```diff
@@ -9,18 +9,54 @@ public class Order
         public string PhoneNumber { get; private set; }
         public decimal TotalPrice { get; private set; }
         public string Comment { get; private set; }
+        public string Email { get; private set; }
         public OrderStatus OrderStatus { get; private set; }
-        public int AddressId { get; private set; }
+        public int? AddressId { get; private set; }
         public Address Address { get; private set; }
-        public int UserId { get; private set; }
+        public int? UserId { get; private set; }
         public User User { get; private set; }
 
+        // Creation fields
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+        public string AddressValue { get; set; }
+        public string City { get; set; }
+        public string PostCode { get; set; }
+
+
         private readonly List<OrderItem> _orderItems = new();
         public IReadOnlyCollection<OrderItem> OrderItems => _orderItems.AsReadOnly();
 
 
         private Order()
         {
         }
+
+        public Order(string email, 
```

Comment:
Order has users and adresses why add this ? An order doen't have a name or a phone number., you can't call an order :D

Reply:
https://github.com/Surinkom-5/PSKApi/pull/3#discussion_r632748746